app/domain/authentication/authn_jwt/fetch_mandatory_claims_to_validate.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnJwt
+    OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
```

Comment:
should this const go in the class itself or is it in the module on purpose? if it's in the module can it go in the consts class?

Reply:
I prefer this const here because it related only to this class logic,
and when we will have another "fetch_mandatory_claims" logic implementation we will have another `OPTIONAL_CLAIMS` value

Reply:
can you please move it then into the class? this way it is part of the module and if you create another one like this in a different class you will see some unexpected behaviour as the other class may use this const.